Userland/Libraries/LibWeb/WebIDL/ExceptionOr.h

```diff
@@ -32,6 +32,8 @@ struct SimpleException {
     Variant<String, StringView> message;
 };
 
+using ExceptionOrCompletion = Variant<SimpleException, JS::NonnullGCPtr<DOMException>, JS::Completion>;
```

Comment:
The completion is considered exceptional here too so I don't think we need "OrCompletion" in the name. We can just call it `WebIDL::Exception` (or `WebIDL::ExceptionType` if there's conflicts, but I don't think there are)